### DISCUSSION TITLE
Error message: 'Cannot read configuration file /usr/local/etc/rtl_airband.conf'

### DISCUSSION THREAD

**Discussion by Amata29** [rtl-airband/RTLSDR-Airband#483] [Q&A]
Hello @ all,
I installed the program V 5.0.0 on Linux Mint using these [https://fuzzthepiguy.tech/rtl_airband-debian/](url) instructions.
By trying to start the program with ''sudo rtl_airband -f'', I get the message: ''Cannot read configuration file /usr/local/etc/rtl_airband.conf''.
What did go wrong? What makes the config file readable? How can I fix this?
Thanks in advance.

**Comment by SullivanChrisJ** [comment]
When using `sudo` to run rtl_airband, as far I can see, the only reason you'd get that message is if the file is not there. Without `sudo`, then using file protection 755 with root ownership works fine for me.

**Comment by Hubdude24** [comment]
Did you fix it? I'm having the same problem

**Comment by SullivanChrisJ** [comment]
@Hubdude24 I see the command used to run was `sudo rtl_airband -f`. Is that what you used? What happens if you use `sudo rtl_airband -f /usr/local/etc/rtl_airband.conf`? Is it any different? What does `ls -l usr/local/rtl_airband.conf` tell you? Which Linux are you using?

**Comment by Hubdude24** [comment]
I used both gave me the same error. I'm on VMware using ubuntu 24

**Comment by SullivanChrisJ** [comment]
There are a number of reasons you can get that error. I looked it up in the code, and here they are.

File Not Found:
The most frequent cause, meaning the specified file path does not point to an existing file. This could be due to a typo in the path, an incorrect relative path, or the file being moved or deleted.

>>> Try entering `cat /usr/local/etc/rtl_airband.conf`. That will tell you if the file is there is not and whether your permissions are correct.

Permissions Issues:
The application or user attempting to read the file lacks the necessary read permissions for that file or its containing directory.

>>> The above command should issue a `Permission denied` error if this is the problem. 

File in Use/Locked:
Another process or application is currently holding a lock on the file, preventing your program from accessing it for reading. This often happens when a file is open in an editor or another program.

>>> Do you have RTLSDR-Airband running already? In the background perhaps - maybe as a system service? You could check by using `ps -ef | grep -I airband`. If the output shows anything other than the `ps -ef` command itself, then it is already running and has the file locked.

Corrupted File or Invalid Data:
The file itself might be corrupted, or its contents might not conform to the expected format, leading to errors during parsing or decoding 

>>> This one is unlikely, especially if the output from `cat /usr/local/etc/rtl_airband.conf` looks OK. If all else fails, you might try recreating the configuration file.

Hardware Issues:
Less common, but underlying hardware problems with the storage device could also lead to I/O errors.

>>> `cat /usr/local/etc/rtl_airband.conf` would most likely expose this one.

Directory as File:
Attempting to read a path that points to a directory instead of a regular file.

>>>The output from `ls -l /usr/local/etc` will show a `d` at the beginning of the line belonging to the file if this is the case.

I've got my money on the file being locked because airband is already running, but I'll be glad to hear which it really was.
